sormas-api/src/main/resources/captions.properties

```diff
@@ -1146,6 +1162,25 @@ Symptoms.unexplainedBleeding = Bleeding or bruising
 Symptoms.unilateralCataracts = Unilateral cataracts
 Symptoms.vomiting = Vomiting
 Symptoms.weight = Weight (kg)
+Symptoms.hydrophobia = Hydrophobia
+Symptoms.opisthotonus = Opisthotonus
+Symptoms.anxietyStates = Anxiety states
+Symptoms.delirium = Delirium
+Symptoms.uproariousness =  Uproariousness
+Symptoms.paresthesiaAroundWound = Paresthesia/Pain around wound
+Symptoms.excessSalivation = Excess salivation
+Symptoms.insomnia = Sleeplessness (Insomnia)
+Symptoms.paralysis = Paralysis
+Symptoms.excitation = Excitation
+Symptoms.dysphagia = Difficulty in swallowing (Dysphagia)
+Symptoms.aerophobia = Fear of flying (Aerophobia)
+Symptoms.hyperactivity = Hyperactivity
+Symptoms.paresis = Paresis
+Symptoms.agitation = Agitation
+Symptoms.ascendingFlaccidParalysis = Ascending flaccid paralysis
+Symptoms.erraticBehaviour = Erratic behaviour
+Symptoms.coma = Coma
+Symptoms.death = Death
```

Comment:
I didn't see a symptom called "Death" (and it wouldn't make sense to have one anyway); remove this